### PR TITLE
Add School Enrollment Details to Driver

### PR DESCRIPTION
**Pull request by zspencer**
Reason for Change
===================
* Allows us to fill the school page with the primary members school information
* Adds some sweet debugging ability while working on prototype pages
* https://trello.com/c/oKZY4ztn/278-drive-the-schools-page

### CODE REVIEW

**Comment by jessieay** [comment]
this seems to include some stuff from `master` -- accidental?

**Comment by zspencer** [comment]
@jessieay  - mmmm, weird. I was trying to get @jayroh's branch up to date with master so it would all merge nice and cleanly together; but I may have screwed it up :(.  Let's not mege this til @jayroh's branch is squashed and merged and then I'll take care of the cleanup.

**Comment by zspencer** [comment]
Never mind, a quick rebase skooshed out the double merge commits.

File: lib/mi_bridges/driver.rb [hunk 1]
```diff
@@ -49,6 +49,11 @@ class Driver
       HousingUtilityBillsSummaryPage,
       YourOtherBillsExpensesPage,
       YourOtherBillsExpensesSummaryPage,
+      SchoolDetailsPage,
+      DebuggerPage,
```
Comment (jessieay) [inline lib/mi_bridges/driver.rb]: do we want to keep these in here?

File: lib/mi_bridges/driver.rb [hunk 2]
```diff
@@ -61,10 +66,15 @@ def run
...
-        page.setup
-        page.fill_in_required_fields
-        page.continue
+        begin
+          page = klass.new(@snap_application)
+          page.setup
+          page.fill_in_required_fields
+          page.continue
+        rescue StandardError => e
+          binding.pry if ENV["DEBUG_DRIVE"]
```
Comment (jessieay) [inline lib/mi_bridges/driver.rb]: nice! do we need this *and* DebuggerPage?

File: lib/mi_bridges/driver/school_details_page.rb [hunk 3]
```diff
@@ -0,0 +1,24 @@
+module MiBridges
+  class Driver
+    class SchoolDetailsPage < ClickNextPage
+      def fill_in_required_fields
```
Comment (jessieay) [inline lib/mi_bridges/driver/school_details_page.rb]: this reads nicely!

File: lib/mi_bridges/driver/school_details_page.rb [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+        choose_enrollment_time_for(snap_application.primary_member)
+        choose_highest_grade_completion_for(snap_application.primary_member)
+      end
+
+      def choose_enrollment_time_for(member)
```
Comment (jessieay) [inline lib/mi_bridges/driver/school_details_page.rb]: I think this can be a private method?

File: lib/mi_bridges/driver.rb (hunk 1 above)
Comment (zspencer) [reply lib/mi_bridges/driver.rb]: @jessieay - These are placeholder pages so you can manually step through the entire process. When we backfill the later pages we will remove them.

File: lib/mi_bridges/driver.rb (hunk 2 above)
Comment (zspencer) [reply lib/mi_bridges/driver.rb]: I think so, this captures weird ass exceptions, whereas debugger page gives you space to fill in the page manually or via the context of a page.

**Comment by adarsh** [comment]
[Drive the "Schools" page](https://trello.com/c/oKZY4ztn/278-drive-the-schools-page)

**Pull request merged**